addons/editor-theme3/addon.json

```diff
@@ -27,8 +27,9 @@
     }
   ],
   "latestUpdate": {
-    "version": "1.31.0",
-    "temporaryNotice": "Two new presets have been added, including a preset for the new high-contrast block colors coming to Scratch soon."
+    "version": "1.33.2",
+    "temporaryNotice": "TODO.",
```

Comment:
We may choose not to add a temporary notice to `editor-theme3` as all the relevant info on it already exists as notices. Note that it will still be at the top of the settings page even if it doesn't have a temporaryNotice.